docs/scenarios.md

```diff
@@ -0,0 +1,53 @@
+# Use Cases
+
+This doc provides sample Ark commands for the following common scenarios:
+* [Disaster recovery][0]
+* [Cluster migration][1]
+
+## Disaster recovery
+
+*Using Schedules and Restore-Only Mode*
```

Comment:
I know this was copy/paste, but I think we should make this section about DR & restore-only mode. I don't think mixing in schedules makes sense, especially since restore-only mode applies to all backups, not just those created by a schedule.

Having a use case that is "here's how you configure a daily backup" makes more sense as a standalone topic.